### DISCUSSION TITLE
Find a partner

### DISCUSSION THREAD

**Discussion by ghost** [leihs/leihs#1673] [Q&A / Community Support]
Good morning,

In the wiki you advise to find a partner to deploy and maintain leihs (https://github.com/leihs/leihs/wiki#support)

Do you have a partner to advise in Switzerland?
Ideally, I am thinking of a partner who already deals with art schools and who would be able to deploy an additional instance, and maintain leihs (update and backup).

Thanks for your advices.

Best regards.
Filippo

**Comment by neoheat** [comment]
Hi @filnug 
yes, there is the partner https://functional.swiss that provides the services you ask for.
best, Lukas
